app/jobs/report_job.rb

```diff
@@ -13,7 +13,12 @@ def done?
   end
 
   def result
-    @result ||= read_result
+    blob = ActiveStorage::Blob.create_and_upload!(io: File.open(filename), filename: filename)
+    ActiveStorage::PurgeJob
+      .set(wait: Rails.configuration.active_storage.service_urls_expire_in)
+      .perform_later(blob)
+    File.unlink(filename)
```

Comment:
Hm, so we still store the file and then upload it. I was hoping to avoid the local storage and upload directly. It means that we need to find a different implementation of ReportJob#done?.